### PR TITLE
Public access to livereload script tag snippet

### PR DESCRIPTION
**Pull request by ataube**
Hey there, 

it would be nice to reuse the livereload script tag snippet (see PR), to extend for instance the connect middleware with some custom functionality. In my case I would like to plugin a dispatcher for app routes to be served with my index.html: 

<pre><code>
var lrScriptTag = require('grunt-contrib-livereload/lib/utils').livereloadScriptTag;

middleware: function (connect, options) {
    var dispatch = require('dispatch');
    var routeDispatcher = function(req, res, next) {
                            
        var index = grunt.file.read(yeomanConfig.app + '/index.html');
        index = index.replace(/&lt;\/body&gt;/, function (w) {
          return lrScriptTag() + w;
        });
        res.write(index);
        res.end();
    }
    return [
        lrSnippet,
        mountFolder(connect, '.tmp'),
        mountFolder(connect, yeomanConfig.app),
        dispatch({
            '/([\\w\\/]+)': routeDispatcher
        })
    ];
}
</code></pre>

Or do you have any better solutions to solve my use case? thanks

### CODE REVIEW

**Comment by sindresorhus** [comment]
What's stopping you from using the lrSnippet middleware?

// @sleeper

File: lib/utils.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ var getSnippet = function () {
   return snippet;
 };
 
+utils.livereloadScriptTag = function livereloadScriptTag() {
+  return getSnippet();
+};
```
Comment (sindresorhus) [inline lib/utils.js]: no need to create another wrapper method, just expose the existing one.

**Comment by ataube** [comment]
Well the lrSnippet middleware handles only html files which are statically located in your app directory. I would also like to support app specific urls for live-reloading  (currently I am getting always a 404 if a specific app url gets reloaded). The following issue describes the problem https://github.com/yeoman/generator/issues/87

File: lib/utils.js (hunk 1 above)
Comment (ataube) [inline lib/utils.js]: ok I see your point, I will remove the wrapper function, any preferred function name?

**Comment by ataube** [comment]
@sindresorhus thanks

**Pull request closed without merging**